core/src/main/scala/scalapb/validate/Validator.scala

```diff
@@ -8,29 +8,39 @@ sealed trait Result {
   def isFailure: Boolean
   def toFailure: Option[Failure]
 
-  def &&(other: => Result) = if (isSuccess) other else this
+  def &&(other: => Result): Result =
```

Comment:
no need to pass by name.

Reply:
Addressed in https://github.com/scalapb/scalapb-validate/pull/19/commits/f438ed35300c7eda697de8b3402479cf55fbf9e0